### DISCUSSION TITLE
Share here ideas about functionality or features that you think would be great to implement into the website

### DISCUSSION THREAD

**Discussion by RandoM02** [ratifire/devrate-service#168] [Ideas]
Напишіть ідеї які ви вважаєте ми могли б імпелементувати у проект. Ми(дизайнери) потім їх відсіємо і можливо ваша ідея буде реалізована. У будь якому випадку ви нам сильно допоможите, якщо пошарите свої ідеї, тому не соромтесь і ділитесь ідеями.

**Comment by MaksymMatveichuk** [comment]
Привіт.

Є пропозиція зробити розділ, де користувач зможе тренувати свої знання перед співбесідою.
Наприклад це буде виглядати у формі карток. На одній стороні питання, на іншій відповідь. Якщо знаєшь, в один кошик, не знаєш в інший на повторення.

**Comment by ovoskresenskyy** [comment]
Achievements
Зробити значки за досягнення. 
Провів 10-50-100 інтерв'ю
5-10-15-20 інтерв'ю з оцінкою вище Х
Та інше
